duo_unix_support/duo_unix_support.sh

```diff
@@ -112,6 +112,14 @@ else
     echo "Could not find version of Duo Unix (login_duo was not found)" > configuration.txt
 fi
 
+if [ "$OS" = "solaris" ]; then
+    if type ggrep >/dev/null; then
+        GREP=ggrep
+    fi
+else
+    GREP=grep
```

Comment:
I don't know my unix enough to know if grep come with most distro but solaris/

Reply:
yeah `grep` is a pretty standard utility. Solaris has a version of `grep` but it doesn't support the same flags as GNU grep.